server/data/draftReportClient.ts

```diff
@@ -94,7 +94,7 @@ export default class DraftReportClient {
   async getDuplicateReports(bookingId: number, [startDate, endDate]: DateRange): Promise<OffenderReport[]> {
     const results = await this.query({
       text: `select r.incident_date date
```

Comment:
now when making a new report it will be incidentLocationId that will be persisted (although locationId may also be persisted if the code change is deployed while a report is partly completed)